ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+* What is the current behavior?
```

Comment:
I've added a prompt to specify the type of issue, and a header that this is not a support forum. Localization issues happen not very often, and I don't want to add too much text (because people often don't read anyway)